Makefile

```diff
@@ -6,9 +6,23 @@ default: moby
 DEPS=$(wildcard cmd/moby/*.go) Makefile
 PREFIX?=/usr/local
 
-moby: $(DEPS)
+moby: $(DEPS) lint
 	go build --ldflags "-X main.GitCommit=$(GIT_COMMIT) -X main.Version=$(VERSION)" -o $@ github.com/moby/tool/cmd/moby
 
+lint:
+	@echo "+ $@: golint, gofmt, go vet"
+	# golint
+	@test -z "$(shell find . -type f -name "*.go" -not -path "./vendor/*" -not -name "*.pb.*" -exec golint {} \; | tee /dev/stderr)"
+	# gofmt
+	@test -z "$$(gofmt -s -l .| grep -v .pb. | grep -v vendor/ | tee /dev/stderr)"
+	# govet
+ifeq ($(shell uname -s), Darwin)
```

Comment:
agree that it's unnecessary for now since we don't have test files - pushed a fix